Ramda/Extensions/has.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  has.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 28/09/2019.
+//
+
+import Foundation
+
+extension R {
+
+    /**
+
+     Returns whether or not an object has an own property with the specified name
+
+     - parameter prop: Property to check
+
+     - returns: True if the property exists; false is it doesn't
+
+     */
+    public class func has<A: Equatable>(_ prop: A) -> ([A: Any]) -> Bool {
```

Comment:
Is there a reason why these functions are curried? We should have separate implementations. A curried and non-curried version of both.

Reply:
I thought about this and it makes sense to keep them separate

Reply:
In the Ramda.js documentation the `has` method shows a curried function 

`const pointHas = R.has(R.__, point)`